src/parser/parser_support.c

```diff
@@ -11669,12 +11830,10 @@ pt_convert_dblink_dml_query (PARSER_CONTEXT * parser, PT_NODE * node, char *sql_
   val->type_enum = PT_TYPE_CHAR;
   val->info.value.string_type = ' ';
 
-  assert (sql_user_text && sql_user_text[0]);
-  if (sql_user_text)
-    {
-      val->info.value.data_value.str = pt_make_remote_query (parser, sql_user_text, snl);
-      PT_NODE_PRINT_VALUE_TO_TEXT (parser, val);
-    }
+  parser->custom_print |= PT_PRINT_SUPPRESS_SERVER_NAME;
```

Comment:
In the `pt_convert_dblink_dml_query` function, `sql_user_text` seems to be no longer needed.
And where the `pt_convert_dblink_merge_query` function is called, it seems that the code below is no longer needed either.

Reply:
Ok, I will fix it.